ethereum-bridged-carbon/package.json

```diff
@@ -12,6 +12,6 @@
   },
   "dependencies": {
     "@graphprotocol/graph-cli": "0.26.0",
-    "@graphprotocol/graph-ts": "^0.27.0"
+    "@graphprotocol/graph-ts": "0.26.0"
```

Comment:
Is this deliberate? Some problem with 0.27.0?

Reply:
Pkbims and I were working in parallel, before pkbims PR was merged, `graph-ts` version was "0.24.1" which was incompatible as it required "0.25+". 

Pkbims updated from "0.24.1 -> 0.27.0"
I updated from "0.24.1 -> 0.26.0" (wanted to have the same version of `graph-cli` and `graph-ts`

When i rebased with pkbims code on `main` branch, i merged conflict by resolving this with current (my) changes (therefore this happened)

I think it does not matter much, i can revert this back to 0.27.0